pysc2/bin/replay_actions.py

```diff
@@ -192,7 +122,7 @@ def run(self):
               self._print("Empty queue, returning")
               return
             try:
-              replay_name = os.path.basename(replay_path)[:10]
+              replay_name = os.path.basename(replay_path)
```

Comment:
changed 10 character truncating to full replay name

Reply:
Okay, I've left the full replay name but I'll leave it up to you to decide on readability. The only catch is that data files are uniquely named from their replay name and will be overwritten if there are colliding names.